pom.xml

```diff
@@ -725,15 +725,20 @@
                 <artifactId>joda-time</artifactId>
                 <version>2.12.7</version>
             </dependency>
+            <dependency>
+                <groupId>junit</groupId>
+                <artifactId>junit</artifactId>
+                <version>4.13.2</version>
+            </dependency>
```

Comment:
@raghavendran-gopalakrishnan I mentioned in standup that I'd noticed `junit` being reintroduced by a dependency build but I couldn't remember which one.  It was here in Job Service.